functions/src/definitions/api/api.ts

```diff
@@ -396,10 +396,35 @@ app.patch("/checkers/:phoneNo/messages/:msgId/voteRequest", async (req, res) =>
 
 })
 
+// TODO: BRENNAN: Complete implementation for not found
```

Comment:
I think the stuff I changed in https://github.com/bettersg/checkMate/pull/240 will have implications here

1. Firstly all the code is using the new checkers collection, no longer using factCheckers (although it'll still be retained). We'll recreate a new checkers collection from the existing factCheckers one, with the new schema.

2. I've made some refactors for the api stuff, namely have moved out the api handler definitions into functions/src/definitions/api/handlers. Eventually I guess we should merge that into this current one, and adjust this accordingly such that each endpoint's handlers are defined in a separate file.

Reply:
Okay, let me look into merging these changes into this branch!